relations/services/model.py

```diff
@@ -20,4 +20,12 @@ class RelationDB(db.Model):
     description = Column(Text(1024))
     added_at = Column(DateTime)
     creator = Column(String(255), nullable=True)
-    supercedes_or_suppresses = Column(Integer, nullable=True)
+    supercedes_or_suppresses = Column(String(255), nullable=True)
+
+
+class ActivationDB(db.Model):
```

Comment:
I'm a bit hesitant to introduce a new table for this. The optimization makes sense, but it creates new data structures that we must always remember to keep in sync with the relations data.

Perhaps an alternative would be to use the foreign key relation for ``supercedes_or_suppresses`` to achieve the same behavior? If the fk is indexed (default in MySQL IIRC), then we should be able to do the ``is_active`` query using that column.

Reply:
Fine, I'll try to do this.